core/app/components/FormBuilder/ElementPanel/InputComponentConfigurationForm.tsx

```diff
@@ -172,7 +188,7 @@ const ComponentSelect = ({
 								onChange(c);
 							}}
 						/>
-						<div className="flex h-[124px] w-full flex-col justify-between rounded-lg border bg-card text-card-foreground shadow-sm peer-checked:border-2 peer-checked:border-ring peer-checked:outline-none">
+						<div className="flex h-32 w-full flex-col justify-between rounded-lg border bg-card text-card-foreground shadow-sm peer-checked:border-2 peer-checked:border-ring peer-checked:outline-none">
```

Comment:
made these boxes a little bigger since otherwise the multivalue input looked super squished inside